.github/workflows/python-package.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     - name: Install Python Dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install pymongo numpy obspy pytest pytest-cov codecov "dask[complete]" schema
+        python -m pip install pymongo numpy obspy pytest pytest-cov dill codecov "dask[complete]" schema
```

Comment:
Need to rebase to current master and add the `dill` module to the new `requirements.txt`.